userspace/falco/application.h

```diff
@@ -0,0 +1,53 @@
+/*
+Copyright (C) 2022 The Falco Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// The falco "app" will eventually replace the monolithic code in
+// falco.cpp. We expect it will be responsible for the following:
+//  - Parsing/validating command line options
+//  - Parsing/validing falco config
+//  - Initialize prerequsites (inspector, falco engine, webserver, etc)
```

Comment:
Always for testing/modularity purposes, I think I would advocate to do dependency injection here (if possible) and provide the code dependencies from outside.

Reply:
Sure, when we get to that we can find a good way to organize it.